digitalpalireader/content/js/conjugate.js

```diff
@@ -0,0 +1,16 @@
+function insertConj() {
```

Comment:
Optional comment: Seem like a candidate for data binding using knockoutjs.

let me know if you are interested. i think databinding is also a long term learning investment. angular/react etc. all use databinding - it is very easy. and i am happy to transfer my knowledge to you.

Reply:
I am interested but not sure if I have time for this.